content/concepts/basics/04-search.md

```diff
@@ -4,17 +4,37 @@ metaTitle: "sensenet - Search"
 metaDescription: "sensenet Search"
 ---
 
-# Search
-https://community.sensenet.com/docs/search-engine/
-https://wiki.sensenet.com/query_syntax.html
-# Full text search
+
+In sensenet one of the most important technical aspects is how we index content items and how we search for them. Especially because sensenet is a query based system which means every content, every list you see on the ui is there because a query found it. In this article we describe the search module and how developers can customize it.
+
 # Metadata indexing, searching by fields
-https://wiki.sensenet.com/Field_Indexing.html
+For every content [link] the field values can be indexed so that when searched for a field value the corresponding content will appear in the result set. It is also possible to search in fields by explicitly defining the field whose values are to be searched within a query. The way a specific field of a content is indexed is defined in the CTD Field Definition.
```

Comment:
```suggestion
For every content [link] the field values can be indexed so that when you search for a term the corresponding content with the related field value will appear in the result set. It is also possible to search by explicitly defining the field whose value we want to search for. The way a specific field of a content is indexed is defined in the CTD Field Definition.
```